Makefile

```diff
@@ -0,0 +1,27 @@
+help:
+	@egrep '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-16s\033[0m %s\n", $$1, $$2}'
+
+lint: ## Lint all python code with flake8
+	flake8 --ignore=E501 denbi/ test
```

Comment:
The linting error appears just with python3.7.

~~~bash
flake8 --ignore=E501 denbi/ test
+denbi/perun/endpoint.py:89:29: W504 line break after binary operator
denbi/perun/endpoint.py:90:29: W504 line break after binary operator
denbi/perun/endpoint.py:91:29: W504 line break after binary operator
denbi/perun/endpoint.py:177:29: W504 line break after binary operator
denbi/perun/endpoint.py:178:29: W504 line break after binary operator
denbi/perun/endpoint.py:179:29: W504 line break after binary operator
~~~

Reply:
Interesting. I don't have a copy of 3.7 on hand. Went ahead and fixed them.